doc/shell_guide.md

```diff
@@ -0,0 +1,880 @@
+
+![shell_linux](./shell_image/image1.png)  
+
+![shell_windows](./shell_image/image2.png)  
+
+If there are crash failures, an alarm icon will appear in the `'Total'` column, like this:  
+
+![shell_crash](./shell_image/image12.png)     
+
+In the overview page, we should check the values in all the colomns. The most important check items are:   
+|column|check point|
+|---|---|
+|Test Rate|Test rate should be 100%.|
+|Fail|How many cases are failed.|
+|Verified Rate|After the verification, verified rate should be 100%.|
+|Elapse Time|The elapse time of the test should not be longer than the elapse time of the previous build too much.|   
+
+### Verfy the Failed Cases
+Click the link in this image to open the online verification page.  
+
+![online_verify1](./shell_image/image3.png)   
+
+On the verification page, click `'Log in'` to login.  
+
+![online_verify2](./shell_image/image4.png)  
+
+After logging in, we can start to verify the failed cases online.    
+
+![online_verify3](./shell_image/image5.png)    
+
+**Click the link in 'Case File' column to check the case and the screen output.**  
+
+![online_verify6](./shell_image/image8.png)  
+
+`'[history]':` check the test case history failure   
+*history page*    
+
+![online_verify4](./shell_image/image6.png)   
+
+`'[verify]':` open the case's verification page to write the verification result   
+*case verification page*   
+
+![online_verify5](./shell_image/image7.png)  
+
+On this page:
+1. select a failure reason.  
+2. write some comments in `'Reason content'`  
+3. write the issue number in `'New issues'` or `'Revise required issues'`  
+4. click the button at the bottom to close this page.  
+`Fix`: I am sure of this verification result.  
+`Save`: I am not sure of this verification result. I will decide it later.  
+
+Sometimes, we cannot find the failure reason from the screen output directly. At this time, we need login the test machine, go to the case path, check the logs. If we still cannot find the failure reason from the logs, we should run the case step by step on this machine to find the failure reason.  
+We should verify the crash failures first.  
+
+### Verify the Low Performance Test Cases
+Open ['QA homepage'](http://qahome.cubrid.org) in browser, then select the build number. `'Function Result'` page will be displayed.   
+Find `'Low Performance Scenarios'` link.  
+
+![online_verify6](./shell_image/image13.png)   
+
+Click this link, and go to the `'Low Performance Scenarios'` page.  
+Click 'SHELL', to check the shell test cases.   
+
+![online_verify7](./shell_image/image14.png)   
+
+We can change the filters, such as `'OS'`, `'Bits'`, `'Category'`, `'Base build'`.  
+Usually, we use the last CI build as base build to compare with.    
+The elapse time of these cases should not changed too much compared to the base build.  
+And a red icon `'+'` will be displayed in `'Flag'` column if the result is abnormal. 

```

Comment:
The read icon means the elaspe time in compared build is under threshold, but on current build, it as new one is great than threshold.